guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -49,21 +57,55 @@ ifdef::satellite[]
+  Corrupted or missing content has been detected, you can examine the list of content in /tmp/unmigratable_content-20211025-74422-16cxfae and take action by either:
+  1. Performing a 'Verify Checksum' sync under Advanced Sync Options, let it complete, and re-running the migration
+  2. Deleting/disabling the affected repositories and running orphan cleanup (foreman-rake katello:delete_orphaned_content) and re-running the migration.
+  3. Manually correcting files on the filesystem in /var/lib/pulp/content/ and re-running the migration
+  4. Mark currently corrupted or missing content as skipped (foreman-rake katello:approve_corrupted_migration_content). This will skip migration of missing or corrupted content.
+----
++
+There are several causes of corrupted or missing content:
+
+  * A Repository synchronization or Content View publish or promotion occurred during the migration.
+  * Files on disk in `/var/lib/pulp/content/` became corrupted due to disk rot.
+  * Files in `/var/lib/pulp/content/` are missing due to past events such as disk loss with a partial restore.
+  * There is some mismatch between subsystems in {Project} such as Katello and Pulp.
+  This can happen if a repository or content view is improperly deleted by skipping steps in a paused {ProjectName} Task.
+  Running `foreman-rake katello:correct_repositories COMMIT=true` can correct this.
++
+You can review the list of corrupt or missing RPMs written to disk as part of the `{foreman-maintain} content migration-stats command`, `/tmp/unmigratable_content-20211025-74422-16cxfae20211120-2149-1j4pmfae` in the example above.
++
+If there are a large percentage of missing or corrupted RPMs, for example, more than 5-10%, {Team} recommends that you review the list generated and contact support for guidance.
++
+In most cases, where the number of missing or corrupt RPMs is low, you can mark these as skipped using the following command:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
-# {foreman-maintain} content prepare
+  # foreman-rake katello:approve_corrupted_migration_content
 ----
 +
+If you approve corrupted or missing content and proceed with the upgrade, that content will not appear in the repositories or content view versions after upgrade.
++
+If these packages exist in the upstream repositories, they are added again when you re-synchronize the repositories after upgrade.
+They are not restored to the Content Views unless you republish those Content Views.
+As these packages are likely unusable, the upgrade process only detects that fact.
++
+If you suspect a Repository synchronization or Content View publish or promotion occurred during the migration, re-running `{foreman-maintain} content prepare` migrates the remaining content.
+If this is the first time you have run `{foreman-maintain} content prepare`, {Team} recommends running it a second time to try to reduce the number of corrupt or missing RPMs.
+Migrating the remaining content then takes less time.
```

Comment:
This appears to suggest you should only run it twice, but the idea is that you can prepare it as often as you'd like. This reduces the migration amount for the real migration and thus shorten the downtime window. So for large installations, it may make sense to run it more than twice, for small installations it may not make any sense.

I don't have a specific suggestion on phrasing, but I hope you can come up with something using this background information.